<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SailfishWizards::Internal::SpectacleProjectTypePage</class>
 <widget class="QWizardPage" name="SailfishWizards::Internal::SpectacleProjectTypePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System package dependences</string>
  </property>
  <property name="title">
   <string>Package components</string>
  </property>
  <property name="subTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select components you want to use in the project:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="componentsTable">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dependenciesDialogButton">
       <property name="text">
        <string>Expert dependencies editor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
